pkg/spec/security.go

```diff
@@ -85,6 +92,10 @@ func (c *SecurityConfig) SetSecurityOpts(runtime *libpod.Runtime, securityOpts [
 		}
 	}
 
+	if !labelSet && !libpodConfig.EnableLabeling {
```

Comment:
It feels like we should be handling this in Libpod - It's the libpod config we're using to set this.

Reply:
Any idea how to detect if an item in `LabelOpts` was set by the user. I moved it here, because we have the info at hand but in libpod we had to be smarter.

Reply:
If `--label` is set while the command line says labeling is disabled, IMO that's an error - SELinux has been disabled, so you can't configure it.

@rhatdan WDYT?

Reply:
I can fix that, then it's straight forward to detect with a simple length check in libpod :+1:

Reply:
Sounds reasonable to me.

Reply:
I begin to believe that `pkg/spec` is the right place for it. We are querying the config there already to lookup certain options. We don't handle the labels really in `./libpod`.